microsoft/botframework-solutions
# [Calendar] add adjust to create calendar

## Description

add adjust to when you say no on create meeting confirm.

## Related Issue





## Testing Steps


go to confirm and say no.
test with
alalala (can not understand)
Time Duration Location and so on
test cancel
test with old flow.

## Checklist

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant

If you have updated responses or `.lu` files:
